include/eve/detail/skeleton.hpp

```diff
@@ -79,12 +83,12 @@ namespace eve::detail
     template<typename Func, typename Idx, typename... Ts>
     EVE_FORCEINLINE auto operator()(Func &&fn, Idx const &i, Ts &&... vs) const noexcept
```

Comment:
why not i by value.

Reply:
because we didn't back in the day, will change.